docker/dev/lemmy.hjson

```diff
@@ -0,0 +1,19 @@
+{
+  # for more info about the config, check out the documentation
+  # https://join-lemmy.org/docs/en/administration/configuration.html
+
+  # This is a minimal lemmy config for the dev / main branch. Do not use for a 
+  # release / stable version.
+
```

Comment:
`hostname` is mandatory to specify, so this wont work.

Reply:
Fixed.